src/ziggurat/init.clj

```diff
@@ -121,13 +135,14 @@
 
 (defn start
   "Starts up Ziggurat's config, reporters, actor fn, rabbitmq connection and then streams, server etc"
-  [actor-start-fn stream-routes actor-routes modes]
+  [actor-start-fn stream-routes batch-routes actor-routes modes]
```

Comment:
Why did we choose the name `stream-routes`? Is it because it abstracts Stream API? If yes, does it make sense to name `batch-routes` and `consumer-routes`?